packages/server-core/src/media/file-browser/file-browser.class.ts

```diff
@@ -370,7 +369,7 @@ export class FileBrowserService
       )
     }
 
-    if (isDev && PROJECT_FILE_REGEX.test(key)) fs.rmSync(path.resolve(projectsRootFolder, key), { recursive: true })
+    if (config.fsProjectSyncEnabled) fs.rmSync(path.resolve(projectsRootFolder, key), { recursive: true })
 
```

Comment:
We seem to have a discrepancy, sometimes we are doing a truthy conditional, other times a strict equality to 'true'.